docs/content/en/docs/clustermgmt/certificate-management/manual-steps-renew-certs.md

```diff
@@ -9,9 +9,9 @@ description: >
   How to rotate certificates for etcd and control plane nodes
 ---
 
-Certificates for external etcd and control plane nodes expire after 1 year in EKS Anywhere. EKS Anywhere automatically rotates these certificates when new machines are rolled out in the cluster. New machines are rolled out during cluster lifecycle operations such as `upgrade`. If you upgrade your cluster at least once a year, as recommended, manual rotation of cluster certificates will not be necessary.
+Certificates for external etcd and control plane nodes expire after 1 year in EKS Anywhere. This page shows the process for manually rotating certificates.
 
-This page shows the process for manually rotating certificates if you have not upgraded your cluster in 1 year.
+Get more information on EKS Anywhere cluster certificates from [here]({{< relref "manual-steps-renew-certs.md" >}})
```

Comment:
why are we referencing the document within itself?

Reply:
Thanks for pointing that out. Updated the page reference to the newly added page.